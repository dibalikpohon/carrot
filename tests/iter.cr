numbers: list = [1,2,3,4,5,6,7,8,9]
iter numbers as num:
	print(num)
end

println()

fruits: list = ["apple", "papaya", "banana"] 
iter fruits as fruit:
	println(fruit)
end
