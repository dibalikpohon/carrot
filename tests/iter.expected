123456789
apple
papaya
banana
